core/extensions.rs

```diff
@@ -15,12 +15,12 @@ pub enum ExtensionFileSourceCode {
   /// inline, or included using `include_str!()`. If you are snapshotting, this
   /// will result in two copies of the source code being included - one in the
   /// snapshot, the other the static string in the `Extension`.
-  IncludedInBinary(&'static str),
```

Comment:
If this file is actually included in the resulting binary I would prefer to keep the old name